gz_ros2_control/package.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0"?>
```

Comment:
Git is treating this as a new file. Can you use `git mv` on the `ign` version of this file so it's treated as a rename?